spec/lib/iiif_print/data/work_files_spec.rb

```diff
@@ -175,8 +181,8 @@
       work = bare_work
       adapter = described_class.of(work)
       adapter.assign(tiff_path)
-      allow(Hyrax::CurationConcern.actor).to receive(:create).and_return(true)
-      expect(Hyrax::CurationConcern.actor).to receive(:create)
+      allow(Hyrax::CurationConcern.actor).to receive(:update).and_return(true)
+      expect(Hyrax::CurationConcern.actor).to receive(:update)
```

Comment:
should this update spec be an addition and not a replacement?

Reply:
Great question, gonna dig in a little more, switching to DRAFT

Reply:
Since you still have create in the method, this may be a good spot to add a context. 

When @work.new_record? is true, test create is called and not update. when false, test update, etc...